services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -155,6 +165,21 @@ async def _get_data_from_port(port: Port) -> Tuple[Port, ItemConcreteValue]:
 
 
 async def download_inputs(inputs_path: Path, port_keys: List[str]) -> ByteSize:
+    return await _download_target_port(
+        port_type_name="inputs", target_path=inputs_path, port_keys=port_keys
+    )
+
+
+async def download_outputs(outputs_path: Path, port_keys: List[str]) -> ByteSize:
+    return await _download_target_port(
+        port_type_name="outputs", target_path=outputs_path, port_keys=port_keys
+    )
```

Comment:
why such fct that call another fct and nothing else? what is the idea?

Reply:
then why not declaring one function
```python
async def download(port_type: Enum[Input,Output], target_path, port_keys)
```?

Reply:
🤷‍♂️ will change to use enum. Fine either way for me.

Reply:
Actually this solves the issue below reported by Pedro pretty nicely.